lib/config/options/index.ts

```diff
@@ -694,6 +694,14 @@ const options: RenovateOptions[] = [
     stage: 'package',
     cli: false,
   },
+  {
+    name: 'useBaseBranchConfig',
+    description:
+      'Read configuration from branches in baseBranches instead of main branch',
+    type: 'string',
+    allowedValues: ['none', 'replace'],
```

Comment:
```suggestion
    allowedValues: ['none', 'merge'],
```

We should change this to merge everywhere